awscli/examples/rum/untag-resource.rst

```diff
@@ -0,0 +1,9 @@
+**To remove one or more tags from the specified resource.**
+
+The following ``untag-resource`` removes one or more tags from the specified resource. If the command succeeds, no output is returned. ::
+
+	aws rum untag-resource \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general